src/hooks/useWindowFitment.ts

```diff
@@ -134,24 +170,36 @@ export const useWindowFitment = (
           fromTop: {
             fitsAbove: referenceTop - heightIncludingSpace > windowTop,
             fitsBelow: referenceTop + heightIncludingSpace < windowBottom,
+            spaceAbove: Math.abs(windowTop - referenceTop),
```

Comment:
I've updated the return value to include the space available, not just whether the item can fit.